Symfony/CS/Tests/Tokenizer/TokensAnalyzerTest.php

```diff
@@ -38,8 +38,14 @@ public function bar4()
 
         return " ({$a})";
     }
+
     public function bar5($data)
     {
+        $message = $data;
+        $example = function ($arg) use ($message) {
+            echo $arg . ' ' . $message;
+        };
+        $example('hello');
```

Comment:
Yes, that would be nice. However the test there is in a diff. file. If this PR is OK'ed I'll open one to update the test at https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tests/Tokenizer/TokensTest.php#L53